apps/client/src/app/components/account-detail-dialog/account-detail-dialog.component.ts

```diff
@@ -133,4 +148,35 @@ export class AccountDetailDialog implements OnDestroy, OnInit {
     this.unsubscribeSubject.next();
     this.unsubscribeSubject.complete();
   }
+
+  private update() {
+    this.isLoadingBenchmarkComparator = false;
+
+    this.dataService
```

Comment:
Can you move up to `ngOnInit()`?

Reply:
@dtslvr Should I relocate the entire function to ngOnInit() or should I call it in ngOnInit()? Is it recommended to eliminate the invocation from the constructor?

Reply:
Yes, move the logic from here to `ngOnInit()`, because there is already logic to fetch other data.